CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing
+
+Thank you for your interest in improving InsightConnect! Please review our [Code of Conduct] before making contributions.
+
+There are multiple ways to contribute beyond writing code. These include:
+
+- [Submit bugs and feature requests] with detailed information about your issue or idea.
+- [Help fellow users with open issues] or [help fellow committers test recent pull requests].
+- [Report a security vulnerability in InsightConnect's plugins] to Rapid7.
+- Submit an updated or brand new plugin!  We are always eager for new
+  integrations or features. Don't know where to start? Check out the [developer documentation].
+
+Here is a short list of do's and don'ts to make sure *your* valuable contributions actually make
+it into production.  If you do not care to follow these rules, your contribution
+**will** be closed. Sorry!
+
+## Code Contributions
+
+- **Do** read the [developer documentation]
+- **Do** stick to the [Python PEP8] and [plugin style] guides.
+- **Do** follow the [50/72 rule] for Git commit messages.
+- **Do** license your code as MIT.
+- **Do** create a [topic branch] to work on. This helps ensure users are aware of commits on the branch being considered for merge, allows for a location for more commits to be offered without mingling with other contributor changes, and allows contributors to make progress while a PR is still being reviewed.
+
+### Pull Requests
+
+- **Do** write "WIP" on your PR and/or open a [draft PR] if submitting **working** yet unfinished code.
```

Comment:
Why does the code have to be working? I would omit that - WIP is totally valid for non-functional, but unfinished code.

Reply:
Example: someone just wants eyes on it or is requesting assistance.

Reply:
Updated